client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -170,6 +173,10 @@ function LineItemWrapper( {
 	const variants = useGetProductVariants( siteId, product.product_slug );
 	const areThereVariants = variants.length > 1;
 
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
+	const isAtomic = useSelector( ( state ) => isAtomicSite( state, siteId ) );
+	const isJetpackSelfHosted = isJetpack && ! isAtomic;
```

Comment:
We might need to also account for Jetpack siteless checkout which probably (?) doesn't get a true value from `isJetpackSite`. We could use `isJetpackPurchasableItem()` on the cart items instead, but I guess that wouldn't find domains. We have `isJetpackCheckout` available in `CheckoutMain` which I believe keeps track of that; that is what was [differentiating the two checkouts before](https://github.com/Automattic/wp-calypso/pull/65374/files). We probably need to pass it down to here again.

Reply:
> but I guess that wouldn't find domains

We don't sell domains to Jetpack, so that should be okay. I'll take a look at both options.

Reply:
In that case, `isJetpackPurchasableItem` might work well.

Reply:
https://github.com/Automattic/wp-calypso/blob/0284a3f23a5e29edd29faf8c192913c52eeb5f93/client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx#L85-L87

Reply:
Thanks - should be handled in 790216c